BookingsApi/BookingsApi.DAL/Commands/UpdateHearingParticipantsCommand.cs

```diff
@@ -74,7 +74,7 @@ public async Task Handle(UpdateHearingParticipantsCommand command)
                     throw new ParticipantNotFoundException(newExistingParticipantDetails.ParticipantId);
                 }
 
-                await _hearingService.RemoveParticipantLinks(participants, existingParticipant);
+                existingParticipant.LinkedParticipants.Clear();
```

Comment:
The hearing service method attempts to remove the linked participants on both the primary and any linked participants. This caused issues when it looked for a linked participant that was already removed.

This works similarly to how the remove participant functionality works.

I made the change here and not in HearingService as that is also used elsewhere by UpdateParticipantCommandHandler which handles changes to 1 participant, I'm not sure if this is actually still used anywhere in admin web other than edit participant name functionality.